decidim-multisurveys/app/models/concerns/decidim/multisurvey/has_questionnaires.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  module Multisurvey
+    # A concern with the components needed when you want a model to be have many questionnaires attached
+    module HasQuestionnaires
```

Comment:
This concern is a copy of the `HasQuestionnaire` concern in Decidim.

In this case, instead of a single questionnaire, it should have many.

I'm not sure if this is necessary, I think it was part of a different solution that consisted of having multiple questionnaires attached to a single component but this was somehow more complex. But maybe I'm wrong and it could be useful.